charts/zookeeper/values.yaml

```diff
@@ -79,3 +82,13 @@ hooks:
 
 containers: []
 volumes: []
+
+## Init containers parameters:
+## volumePermissions: Change the owner of the persistent volume mountpoint to RunAsUser:RunAsGroup
+
+volumePermissions:
+  enabled: false
+  #initContainer:
+    #name: initcontainer
```

Comment:
I wonder if giving an initContainer.name field here adds anything.   Since that specific init container is actually fully managed by the operator, maybe hardcode/set a name for it in the code, no matter what, like you are doing at line 190 of generators.go. but maybe something more meaningful like volume-perms-init or similar.

Reply:
Done